src/api/idea-card/controllers/idea-card.js

```diff
@@ -1,9 +1,21 @@
-'use strict';
+"use strict";
+
+const { type } = require("os");
 
 /**
  *  idea-card controller
  */
 
-const { createCoreController } = require('@strapi/strapi').factories;
+const { createCoreController } = require("@strapi/strapi").factories;
 
-module.exports = createCoreController('api::idea-card.idea-card');
+module.exports = createCoreController(
+  "api::idea-card.idea-card",
+  ({ strapi }) => ({
+    async findOne(ctx) {
```

Comment:
Could this be a global middleware? This problem can happen to any endpoint.

Reply:
Yep, but let do it in a separate PR